packages/sdk/test/crdt_pbt/counter_test.ts

```
@@ -80,11 +100,21 @@ function counterTraceArbitrary(
     });
 }
 
+function expectedCounterJSON(trace: Array<CounterStep>): string {
+  let sum = 0n;
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)